wolfcrypt/src/ed448.c

```diff
@@ -402,8 +483,10 @@ int wc_ed448ph_sign_msg(const byte* in, word32 inLen, byte* out, word32 *outLen,
  *          other -ve values when hash fails,
  *          0 otherwise.
  */
-int wc_ed448_verify_msg_init(const byte* sig, word32 sigLen, ed448_key* key,
-                        byte type, const byte* context, byte contextLen)
+
+static int wc_ed448_verify_msg_init_1(const byte* sig, word32 sigLen,
```

Comment:
Is `_1` the best name? Not clear to me the difference.

Reply:
`_1` is a de facto standard for static internal implementations -- don't we already use that all over the place in the library?  need to check.

Reply:
checked.  the practice isn't common in wolfSSL but it (pre)exists.  it's somewhat common in the Linux kernel.

do we have a different de facto standard way of naming routines for this sort of thing?

Reply:
done, renamed with `_with_sha` suffixes, and without the `wc_` prefix.